js/src/forum/components/PostStreamScrubber.js

```diff
@@ -133,6 +136,15 @@ export default class PostStreamScrubber extends Component {
       .on('mouseup touchend', (this.handlers.onmouseup = this.onmouseup.bind(this)));
 
     setTimeout(() => this.scrollListener.start());
+
+    this.updateScrubberValues({ animate: true, forceHeightChange: true });
```

Comment:
This does not wait for the `loadPromise`, as does `onupdate`. I'd expect that to be even more relevant on first render, but may be wrong...

Reply:
Interesting, iirc, this was more of a "it works this way" implementation. That begs the question: Should other calls wait for the promise? If so, should that be a part of the updateScrubberValues method itself?

Reply:
Let's not try figuring that out here. :wink:  Since it's a time-based thing, it may also be a problem only under certain conditions, or never. I just noticed it's technically a change in behavior.

Reply:
Moved to https://github.com/flarum/core/issues/2297